client/src/components/projects/project-filter/ProjectFilter.vue

```diff
@@ -23,36 +23,39 @@
   import {IProject} from '../../../shared/interfaces/IProject';
   import Accordion from '../../common/Accordion/Accordion.vue';
   import Checkbox from '../../common/Checkbox/Checkbox.vue';
-  import Constants from '../../../shared/constants/project.constants';
   import {IProjectFilter, IProjectFilterCheck} from './IProjectFilter';
-
+  import * as types from '../../../store/modules/projects/project-types';
+  import {IModel} from '../../../shared/interfaces/IModel';
+  import {IProgram} from "../../../shared/interfaces/IProgram";
 
   export default Vue.extend({
       data() {
         return {
-          models: new Array<IProjectFilter>()
+          models: [] as IProjectFilter[]
         }
       },
       created() {
-        // TODO make it in computed property after moving constants to db
-        this.models = Object.keys(Constants).map(key => {
-          const model: IProjectFilter = {
-            name: key,
-            opened: true,
-            items: this.createModelForCheckboxes(Constants[key])
-          };
+        this.$store.dispatch(types.FETCH_ADDONS);
+      },
+      computed: {
 
-          return model;
-        });
+        modelsComputed(): IProjectFilter[] {
+          const addons = this.$store.getters[types.GET_ADDONS];
+          this.models = [] as IProjectFilter[];
 
-        const customers: IProjectFilter = {
-          name: 'Customers',
-          opened: true,
-          items: this.createModelForCheckboxes(this.$store.state.projects.customers)
-        };
-        this.models.push(customers);
-      },
+          Object.keys(addons).forEach(key => {
+            const model: IProjectFilter = {
+              name: key,
+              opened: true,
+              items: this.createModelForCheckboxes(addons[key])
+            };
 
+            this.models.push(model);
+          });
+
+          return this.models;
```

Comment:
Again - remove data and just return some array

Reply:
doesn't work without data, checkboxes won't update accordion won't toggle